packages/core/src/Provider/NetworkInfoProvider/types.ts

```diff
@@ -3,7 +3,7 @@ import { Cardano, TimeSettings } from '../..';
 export type ProtocolParametersRequiredByWallet = Required<
   Pick<
     Cardano.ProtocolParametersAlonzo,
-    | 'coinsPerUtxoWord'
```

Comment:
It's [used](https://github.com/input-output-hk/cardano-js-sdk/blob/master/packages/cip2/src/selectionConstraints.ts#L44) when computing minimum coin quantity. Is this just a rename or is it the `perByte` value? We might need to also adjust the computation.